### ISSUE TITLE
godot.InputEvent pressed property doesn't work, but is_pressed getter does

### ISSUE THREAD

**Issue by MichaelBelousov** [GodotExplorer/ECMAScript#48]
Here's a reproduction:
[ecmascript-pressed.zip](https://github.com/GodotExplorer/ECMAScript/files/5079860/ecmascript-pressed.zip)

Here's the running script:

```ts
export default class A extends godot.Node2D {
  _input(evt) {
    if (evt.button_index === godot.ButtonList.BUTTON_RIGHT && evt.is_pressed())
      godot.print('right click!')
    if (evt.button_index === godot.ButtonList.BUTTON_LEFT && evt.pressed)
      godot.print('left click!')
  }
}
```

Notice that the right click triggers the print, but not the left.

**Comment by Geequlim** [comment]
Does it works on GDScript ?

**Comment by MichaelBelousov** [comment]
will confirm but it should

**Comment by MichaelBelousov** [comment]
It does, I converted it like so:

```gdscript

extends Node2D

class_name A

func _input(evt):
  if evt.button_index == BUTTON_RIGHT and evt.is_pressed():
    print('right click!')
  if evt.button_index == BUTTON_LEFT && evt.pressed:
    print('left click!')
```

**Comment by Geequlim** [comment]
You'd better report in upstream why this is commented out.
https://github.com/godotengine/godot/blob/72d12289bb5e46ad2ec840a23a247f5690afb0d3/core/os/input_event.cpp#L539

**Comment by MichaelBelousov** [comment]
great question...

**Comment by MichaelBelousov** [comment]
the git blame says it was done 3 years ago when removing the input event from the variant type. I can try to find the original author of that change on discord and ask them

**Comment by MichaelBelousov** [comment]
I haven't gotten a reply by private message, I'll ask in one of the chats and eventually move to an issue on the github if no one can answer.

**Comment by MichaelBelousov** [comment]
On IRC Fales (the godot JavaScript backend maintainer) told me that is commented out because it is bound in the super class:
```irc
fales[m]
17:38:03
mikemikeb: I think because it's exposed in the superclass "InputEvent".
https://github.com/godotengine/godot/blob/72d12289bb5e46ad2ec840a23a247f5690afb0d3/core/os/input_eve...
```

**Comment by Geequlim** [comment]
Ok I will find time to fix this  soon.

**Comment by Geequlim** [comment]
Thanks for your report. This issue is fixed by bedb532
